declarations/dom.d.ts

```diff
@@ -0,0 +1,7 @@
+interface Element {
+    requestPointerLock(): void;
+}
+
+interface Document {
+  pointerLockElement: Element;
+}
```

Comment:
Apparently this is the recommended fix.

https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#recommendations